content/applications/sales/subscriptions/products.rst

```diff
@@ -1,61 +1,106 @@
-==================
-Recurring products
-==================
+===============================
+Configure subscription products
+===============================
```

Comment:
Hi Melanie :)

Please avoid unnecessary verbs (such as configure, manage, create, etc) in your titles and headings. It is easier to navigate the doc menu if the titles are concise. ([see guidelines](https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html#titles-and-headings)). Of course, it's sometimes very useful to have a verb in the title, but here the noun group "Subscription products" works as well.

```suggestion
=====================
Subscription products
=====================
```

I haven't read the rest of the PR, it was just a small comment about the title. :)

Have a good day!